src/storage/azure.js

```diff
@@ -50,6 +50,34 @@ const downloadFileAsJson = ({ blobServiceClient }) => async (
   }
 };
 
+const downloadFileAsString = ({ blobServiceClient }) => async (
```

Comment:
use this function for `downloadFileAsJson` ? it's basically calling `downloadFileAsString` and then `JSON.parse`